cpp/velox/compute/VeloxBackend.cc

```diff
@@ -272,6 +284,30 @@ void VeloxBackend::initConnector(const std::shared_ptr<const facebook::velox::Co
   }
 #endif
 
+  mutableConf->setValue(
+      velox::connector::hive::HiveConfig::kEnableFileHandleCache,
+      conf->get<bool>(kVeloxFileHandleCacheEnabled, kVeloxFileHandleCacheEnabledDefault) ? "true" : "false");
+
+  mutableConf->setValue(
+      velox::connector::hive::HiveConfig::kMaxCoalescedBytes,
+      conf->get<std::string>(kMaxCoalescedBytes, "67108864")); // 64M
+  mutableConf->setValue(
+      velox::connector::hive::HiveConfig::kMaxCoalescedDistanceBytes,
+      conf->get<std::string>(kMaxCoalescedDistanceBytes, "1048576")); // 1M
+  mutableConf->setValue(
+      velox::connector::hive::HiveConfig::kPrefetchRowGroups, conf->get<std::string>(kPrefetchRowGroups, "1"));
+  mutableConf->setValue(
+      velox::connector::hive::HiveConfig::kLoadQuantum, conf->get<std::string>(kLoadQuantum, "268435456")); // 256M
+  mutableConf->setValue(
+      velox::connector::hive::HiveConfig::kFooterEstimatedSize,
+      conf->get<std::string>(kDirectorySizeGuess, "32768")); // 32K
+  mutableConf->setValue(
+      velox::connector::hive::HiveConfig::kFilePreloadThreshold,
+      conf->get<std::string>(kFilePreloadThreshold, "1048576")); // 1M
+
+  // set cache_prefetch_min_pct = 0 to force all loads are prefetched in DirectBufferInput.
+  FLAGS_cache_prefetch_min_pct = 0;
```

Comment:
Could you help add gluten conf for this property also instead of hard code 0 here?

Reply:
Added, thank you